commands/show_cluster.go

```diff
@@ -187,6 +233,21 @@ func showClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 	}
 	output = append(output, color.YellowString("Created:")+"|"+util.ShortDate(created))
 	output = append(output, color.YellowString("Organization:")+"|"+clusterDetails.Owner)
+
+	if credentialDetails != nil && credentialDetails.ID != "" {
+		if credentialDetails.Aws != nil {
+			parts := strings.Split(credentialDetails.Aws.Roles.Awsoperator, ":")
+			if len(parts) > 3 {
+				output = append(output, color.YellowString("AWS account:")+"|"+parts[4])
+			} else {
+				output = append(output, color.YellowString("AWS account:")+"|n/a")
+			}
+		} else if credentialDetails.Azure != nil {
+			output = append(output, color.YellowString("Azure subscription:")+"|"+credentialDetails.Azure.Credential.SubscriptionID)
+			output = append(output, color.YellowString("Azure tenant:")+"|"+credentialDetails.Azure.Credential.TenantID)
+		}
+	}
```

Comment:
Credentials are printed in `sumWorkerCPUs`?

Reply:
No, in `showClusterRunOutput`

Reply:
Oh sorry I didn't spot another fold.